src/User32.Desktop/User32+SPI.cs

```diff
@@ -0,0 +1,1315 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SPI"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// SPI_ System-wide parameter : Used in SystemParametersInfo function
+        /// </summary>
+        public enum SPI : uint
```

Comment:
As the enum itself doesn't exists in the original API we can give it a more expressive name (I don't know what name as `SystemParametersInfo` would clash with the function